modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/UpgradeLayoutFeatures.kt

```diff
@@ -145,12 +179,13 @@ internal fun UpgradeLayoutFeatures(
 @Composable
 private fun SubscribeButton(
     onClickSubscribe: () -> Unit,
+    bottomPadding: Dp,
```

Comment:
It might make the component more reusable if we don't pass in the bottom padding and instead use a modifier.
```kotlin
@Composable
private fun SubscribeButton(
    onClickSubscribe: () -> Unit,
    modifier: Modifier = Modifier,
) {
    Box(
        contentAlignment = Alignment.BottomCenter,
        modifier = modifier,
    ) {
        RowButton(
            modifier = Modifier.padding(horizontal = 16.dp),
            text = stringResource(LR.string.get_pocket_casts_plus),
```

Reply:
[82e4d2b](https://github.com/Automattic/pocket-casts-android/pull/2674/commits/82e4d2b79562bcb230e48d386ae5657317736b11)